### ISSUE TITLE
WCAG 2.0 Some buttons missing names describing purpose

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/fgpv-vpgf#3811]
The more options buttons for the layers on the map do not have names that describe their purpose. Users depending on the text on these buttons will not know their purpose. This does not meet WCAG success criteria:

WCAG 2.0: 1.1.1 Non-text content
https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html
WCAG 2.1: 2.5.3 Label in name
https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html
WCAG 2.0: 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
Recommended fix. Make sure that all buttons on the map have names that concisely describes their purpose.
